scripts/load_config.go

```diff
@@ -0,0 +1,31 @@
+package scripts
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/Brightscout/mattermost-load-test-scripts/serializers"
+)
+
+// TODO: remove later, if not needed
+func LoadConfig() {
+	configFile, err := os.Open("config/config.json")
+	if err != nil {
+		panic(err)
+	}
+
+	defer configFile.Close()
+	byteValue, err := ioutil.ReadAll(configFile)
+	if err != nil {
+		panic(err)
+	}
+
+	var config serializers.Config
+	if err := json.Unmarshal(byteValue, &config); err != nil {
+		panic(err)
+	}
+
+	fmt.Println(config.ConnectionConfiguration.ServerURL)
```

Comment:
need for this statement ?

Reply:
there is already a TODO comment above the function to remove it if not needed. also, we have removed this from next PR